src/components/delegate-tag/components/delegate-tag-container/delegate-tag-container.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from 'react';
+import { View } from 'react-native';
+
+import { TokenInterface } from '../../../../token/interfaces/token.interface';
+import { getTokenSlug } from '../../../../token/utils/token.utils';
+import { useApyStyles } from '../../../token-screen-content-container/apy.styles';
+import { useDelegateTagContainerStyles } from './delegate-tag-container.styles';
+
+interface DelegateTagContainerProps {
+  token: TokenInterface;
+}
+
+export const DelegateTagContainer: FC<DelegateTagContainerProps> = ({ token, children }) => {
+  const styles = useDelegateTagContainerStyles();
+  const apyStyles = useApyStyles();
+
+  const tokenSlug = getTokenSlug(token);
+
+  return children !== false ? <View style={[styles.apyContainer, apyStyles[tokenSlug]]}>{children}</View> : null;
```

Comment:
```suggestion
  return children && <View style={[styles.apyContainer, apyStyles[tokenSlug]]}>{children}</View>;
```
However, it may be better not to apply this suggestion but to implement hiding this component outside of it

Reply:
I think we can't use it because && operator returns a boolean instead of react-element. alternative we may to use <>{children && ...}</>